pkg/kubelet/status/manager.go

```diff
@@ -182,16 +186,37 @@ func (m *manager) TerminatePods(pods []*api.Pod) bool {
 				Terminated: &api.ContainerStateTerminated{},
 			}
 		}
-		select {
-		case m.podStatusChannel <- podStatusSyncRequest{pod, pod.Status}:
-		default:
+		newStatus := m.updateStatusInternal(pod, pod.Status)
```

Comment:
That should be fine - my primary concern originally was not introducing more change than necessary in status, and fixing that is definitely desirable.